can-deparam-test.js

```diff
@@ -36,6 +36,10 @@ test('Remaining ampersand', function () {
 		}
 	});
 });
+test('Invalid encoding', function() {
+	deparam('foo=%0g');
+	ok(true, 'Did not throw');
```

Comment:
Rather than just asserting that it doesn't throw, I think this test should verify that the return value of deparam() is what it should be.